app/webhook.go

```diff
@@ -323,13 +323,9 @@ func (a *App) CreateIncomingWebhookForChannel(creatorId string, channel *model.C
 		return nil, model.NewAppError("CreateIncomingWebhookForChannel", "api.incoming_webhook.invalid_username.app_error", nil, "", http.StatusBadRequest)
 	}
 
-	if result := <-a.Srv.Store.Webhook().SaveIncoming(hook); result.Err != nil {
-		return nil, result.Err
-	} else {
-		return result.Data.(*model.IncomingWebhook), nil
-	}
-}
+	return a.Srv.Store.Webhook().SaveIncoming(hook)
 
```

Comment:
nit: I would expect this blank line right after the `}` and no blank line between `return` statement and `}`